### PR TITLE
templates: CMS Physics Object -pages

### LINKED ISSUE #1370: templates: /about/CMS-Physics-Objects
Implement http://opendata.cern.ch/about/CMS-Physics-Objects/

- case 2 - Page
- body in markdown

### LINKED ISSUE #1371: templates: /about/CMS-Physics-Objects/2010
Implement http://opendata.cern.ch/about/CMS-Physics-Objects/2010

Implement anchors like http://opendata.cern.ch/about/CMS-Physics-Objects/2011#muon2011

Body in markdown

### LINKED ISSUE #1372: templates: /about/CMS-Physics-Objects/2011
Implement http://opendata.cern.ch/about/CMS-Physics-Objects/2011

Implement anchors like http://opendata.cern.ch/about/CMS-Physics-Objects/2011#muon2011

Body in markdown

### PR DESCRIPTION
**Pull request by hjhsalo**
* Adds `articles/cms-physics-objects-2010` and `articles/cms-physics-objects-2011`
  (closes #1371)
  (closes #1372)
* Adds redirects from `/cms-physics-objects/<year>` to corresponding physics-objects article.
  If no year is given redirects to latest available physics-objects page (currently 2011)
  (closes #1370)

Signed-off-by: Harri Hirvonsalo <harri.hirvonsalo@cern.ch>

### CODE REVIEW

File: cernopendata/modules/pages/views.py [hunk 1]
```diff
@@ -86,6 +86,79 @@ def index():
...
+                           content=f.read().decode("utf-8"))
+
+
+@blueprint.route('/education')
+@blueprint.route('/education/<string:experiment>')
+@register_breadcrumb(blueprint, '.education.experiment',
+                     lazy_title('%(experiment)s', 'experiment'),
+                     endpoint_arguments_constructor=lambda: {
+                         'experiment': request.view_args['experiment']})
+def education(experiment=None):
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/pages/views.py]: remove these blueprints and the whole function as we are taking care of them in the faceted search.

File: cernopendata/modules/pages/views.py [hunk 2]
```diff
@@ -86,6 +86,79 @@ def index():
...
+
+
+@blueprint.route('/research')
+@blueprint.route('/research/<string:experiment>')
+@register_breadcrumb(blueprint, '.research.experiment',
+                     lazy_title('%(experiment)s', 'experiment'),
+                     endpoint_arguments_constructor=lambda: {
+                         'experiment': request.view_args['experiment']})
+def research(experiment=None):
+    """Display research pages."""
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/pages/views.py]: Same here.

File: cernopendata/modules/pages/views.py (hunk 1 above)
Comment (hjhsalo) [reply cernopendata/modules/pages/views.py]: Accidentally included, sorry.

File: cernopendata/modules/pages/views.py (hunk 2 above)
Comment (hjhsalo) [reply cernopendata/modules/pages/views.py]: Accidentally included, sorry.

**Comment by hjhsalo** [comment]
@ioannistsanaktsidis 
Both comments fixed. I'm not sure in which rebase I managed to catch those things, so could you check that I haven't introduced any other obsolete stuff? Would do it myself, but not sure how...

File: cernopendata/modules/fixtures/data/articles/cms-physics-objects-2010/cms-physics-objects-2010.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+1. Quality flags: MET should be cleaned for detector problems; there should be several flags for these
+2. Type-I: whether or not JEC is propagated to MET, the JEC is an important systematic
+3. Important uncertainties are JER, JES and unclustered energy: the JER resmearing (uncertainty on JER per jet: 5–20%) and JES systematics should be propagated to MET, and unclustered energy should be considered as an extra systematic by scaling it up and down by 10%.
+4. For SUSY-type analysis, MET tails often come from jets (ECAL holes, heavy flavour, JER), so analyses typically apply cuts on Δφ(jet, MET) to ensure MET is not aligned with one of the jets
+
+<br>
+
+
+## <div id="bjet2011"> b-jets <img src="../../../../static/articles/cms-physics-objects-2011/jet_b.png" width="10%" align="right"> </div>
```
Comment (atrisovic) [inline cernopendata/modules/fixtures/data/articles/cms-physics-objects-2010/cms-physics-objects-2010.md]: I think pictures can work without `../../..` :) you can only say `/static/path`
I changed it in the more recent files

Comment (hjhsalo) [reply cernopendata/modules/fixtures/data/articles/cms-physics-objects-2010/cms-physics-objects-2010.md]: OK

Comment (hjhsalo) [reply cernopendata/modules/fixtures/data/articles/cms-physics-objects-2010/cms-physics-objects-2010.md]: Fixed.

**Pull request merged**